scale/scale/local_settings_docker.py

```diff
@@ -19,21 +19,36 @@
 STATIC_ROOT = os.environ.get('SCALE_STATIC_ROOT', 'static/')
 STATIC_URL = os.environ.get('SCALE_STATIC_URL', '/scale/static/')
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.contrib.gis.db.backends.postgis',
-        'NAME': os.environ.get('SCALE_DB_NAME', ''),
-        'USER': os.environ.get('SCALE_DB_USER', ''),
-        'PASSWORD': os.environ.get('SCALE_DB_PASS', ''),
-        'HOST': os.environ.get('SCALE_DB_HOST', ''),
-        'PORT': os.environ.get('SCALE_DB_PORT', '5432'),
-    },
-}
+DB_HOST = os.environ['SCALE_DB_HOST']
+if DB_HOST == '':
+        DB_HOST = os.environ.get('DB_PORT_5432_TCP_ADDR', '')
```

Comment:
If you use docker --link that's how the link addresses show up. It's used by the vagrant image